src/string_enums/ai_target.hpp

```diff
@@ -0,0 +1,34 @@
+/*
+	Copyright (C) 2008 - 2021
```

Comment:
2021

Reply:
Since this is adding a bunch of new files, I figured I'd run the copyright update script on the repo after merging this.